apps/cowswap-frontend/src/modules/orderProgressBar/hooks/useOrderProgressBarProps.ts

```diff
@@ -312,7 +316,11 @@ function useCancellingOrderUpdater(orderId: string, isCancelling: boolean): void
   const setCancellationTriggered = useSetAtom(setOrderProgressBarCancellationTriggered)
 
   useEffect(() => {
-    if (isCancelling) setCancellationTriggered(orderId)
+    if (!orderId || !isCancelling) {
```

Comment:
nitpick: the same question about undefined case

Reply:
https://github.com/cowprotocol/cowswap/pull/6511/commits/47eb27e15c993604417c0784994a01a60ce356ed